Obsidian/Program.cs

```diff
@@ -58,66 +58,44 @@ private static async Task Main()
                 Console.CancelKeyPress += OnConsoleCancelKeyPressed;
             }
 
-            var globalConfigFile = new FileInfo(Program.globalConfigFile);
+            string serverDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
This shouldn't use the binary path in any way, a relative path should be used instead because .NET will always resolve it to the current working directory.

Consider changing it to a `string.Empty` for now until we might have to change it for something else.